partners/scientific-communities.md

```diff
@@ -1,4 +1,4 @@
-# pyOpenSci Software Peer Review Partnerships
+# Scientific Python Partnerships
```

Comment:
Scientific Python here means https://scientific-python.org/ ?

Reply:
ahhh no i mean as in we partner with scientific Python communities such as astropy. so lower case "s" rather than the project. maybe i could write:

Partnerships with Scientific Python communities?

would that be more clear?

Reply:
ok i follow. so perhaps just

pyOpenSci Community Peer Review Partnerships - closer to how it was initially is best?